community/cypher/runtime-util/src/main/scala/org/neo4j/cypher/internal/runtime/planDescription/LogicalPlan2PlanDescription.scala

```diff
@@ -33,9 +33,10 @@ object LogicalPlan2PlanDescription extends ((LogicalPlan, PlannerName) => Intern
                      plannerName: PlannerName): InternalPlanDescription = {
     val readOnly = input.solved.readOnly
     new LogicalPlan2PlanDescription(readOnly).create(input)
-      .addArgument(Version("CYPHER 3.4"))
+      .addArgument(RuntimeVersion("CYPHER 3.4"))
```

Comment:
Before the Version related to both planner and runtime, and the text `Cypher x.y` would imply an `x.y` version of the Cypher language (parsing, planning, executing). Now we are talking here only about the runtime, but in fact from the users perspective, when they see `Cypher x.y` they are thinking about the version of the language spec, which is tied to the parser and planner much more than the runtime.

I think this text is also going to come out in the query plan description, and we will see the wrong values here. For example, if the user runs `PROFILE CYPHER 3.3 RETURN 1`, they are going to see a plan description with `CYPHER 3.4` printed, instead of the version they expected.

Reply:
I will change the toString of PlanDescription prefix to account for both plannerVersion and runtimeVersion. This will account for both this and the above comment.

Reply:
As of the latest commit, the output of the planDescription of `CYPHER 3.3 EXPLAIN MATCH (n) RETURN n` is

```
Planner COST 3.3

Runtime INTERPRETED CYPHER 3.4

+-----------------+----------------+-----------+
| Operator        | Estimated Rows | Variables |
+-----------------+----------------+-----------+
| +ProduceResults |              1 | n         |
| |               +----------------+-----------+
| +AllNodesScan   |              1 | n         |
+-----------------+----------------+-----------+
```

I agree that having "CYPHER" in the runtime version vs. not having it in the planner version is kinda weird.

Reply:
Also, with that latest commit, the browser still displays "Cypher version: CYPHER 3.4, planner: COST, runtime: INTERPRETED" for the above EXPLAIN query...
Probably we need to change the interaction between the two here.